ziti/edge/impl/listener.go

```diff
@@ -114,7 +114,7 @@ func (listener *edgeListener) updateCostAndPrecedence(cost *uint16, precedence *
 	logger.Debug("sending update bind request to edge router")
 	request := edge.NewUpdateBindMsg(listener.edgeChan.Id(), listener.token, cost, precedence)
 	listener.edgeChan.TraceMsg("updateCostAndPrecedence", request)
-	return listener.edgeChan.SendWithTimeout(request, 5*time.Second)
+	return request.WithTimeout(5 * time.Second).SendAndWaitForWire(listener.edgeChan.Channel)
```

Comment:
all these 5 * time.second seem like they could be a constant or a function or 'anything else' maybe?